src/models/Job.js

```diff
@@ -108,34 +98,35 @@ module.exports = (sequelize) => {
         type: Sequelize.STRING(255),
         allowNull: false
       },
-      createdAt: {
-        field: 'created_at',
-        type: Sequelize.DATE,
-        allowNull: false
-      },
       createdBy: {
         field: 'created_by',
         type: Sequelize.UUID,
         allowNull: false
       },
-      updatedAt: {
-        field: 'updated_at',
-        type: Sequelize.DATE
-      },
       updatedBy: {
         field: 'updated_by',
         type: Sequelize.UUID
-      },
-      deletedAt: {
-        field: 'deleted_at',
-        type: Sequelize.DATE
       }
     },
     {
       schema: config.DB_SCHEMA_NAME,
       sequelize,
       tableName: 'jobs',
-      timestamps: false
+      paranoid: true,
+      deletedAt: 'deletedAt',
+      createdAt: 'createdAt',
+      updatedAt: 'updatedAt',
+      timestamps: true,
```

Comment:
timestamps must be enable in favor of paranoid mode. It also means `updatedAt` will be automatically set when creating a record.

Reply:
@imcaizheng I thought this would not be required as documentation didn't mention this. But if we need to enable it this is fine. Actually, I was considering if we have to switch to utilizing `timestamps` options instead of manual settings these values but didn't find strong reasons.

The only thing, as we enable `timestamps` we should remove all the code which sets `createdAt` / `updatedAt` dates manually, to keep things clear when we update https://github.com/topcoder-platform/taas-apis/blob/dev/src/services/JobService.js#L202 or create some data https://github.com/topcoder-platform/taas-apis/blob/dev/src/services/JobService.js#L153.

Reply:
@maxceem All the redundant code is removed now.